x-pack/plugins/infra/public/common/visualizations/lens/dashboards/asset_details/host/host_kpi_charts.ts

```diff
@@ -10,6 +10,8 @@ import { hostLensFormulas } from '../../../formulas';
 import { METRICS_TOOLTIP } from '../../translations';
 import type { KPIChartProps } from '../../types';
 
+const BACKGROUND_COLOR = '#e6f1fA';
```

Comment:
Shouldn't we use the euiTheme for this `euiTheme.eui.euiColorLightestShade`?

APM has a similar view and it uses this color:
![Uploading Screenshot 2023-11-21 at 14.11.41.png…]()

Reply:
I like your idea but this color was part of the design of this issue (linked in the PR) and that was the reason to use it. Also the suggested color looks a bit too close the the background I think so probably that's one of the reasons to consider light blue. 
![image](https://github.com/elastic/kibana/assets/14139027/10862557-a2d7-4ac1-9d54-ed5639177716)

I tried to find the same color in the [eui background colors](https://eui.elastic.co/#/theming/colors/utilities#background-colors) and tried `useEuiBackgroundColor('primary')` but then the dark mode looks off: 
![image](https://github.com/elastic/kibana/assets/14139027/c7e3db8e-194d-4ce7-aa27-a322c0f6b1a9)

AFAIK this color was picked based on feedback given but If we want to change the color I can discuss this again with @kkurstak if needed.

Reply:
IMHO I don't think we should hard-code a color in the code, we should use what is provided by the theme, to avoid inconsistency across the plugins. Maybe it would be a good idea to chat with someone from the Design team indeed.

Reply:
I agree with you, it makes more sense to be consistent. But I am not the designer of this feature so I can't really decide what looks best so I will add 2 versions to the issue: 
- Using APM colors (light & dark mode)
- Using EUI primary background (light & dark mode)
Based on that I can change it with one of the provided by eui styles based on the feedback there.

Reply:
let's what @kkurstak has to say about it.

Reply:
As we discussed in the issue I changed the color to `euiTheme.eui.euiColorLightestShade` to be consistent with APM. @cauemarcondes thank you for the suggestion!